CHANGELOG.md

```diff
@@ -18,12 +20,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Rewrote "cannot find field for current simulation time" error message for clarity
 - Updated `HCO_Error` and `HCO_Warning` to call `HCO_WordWrapPrint` to wrap messages to 78 characters wide
 - Replaced terse error messages in `hcoio_read_std_mod.F90` with more descriptive ones
+- Updated error messages in routine `ExtStateInit` to be more descriptive
+- Changed call to `ExtDat_Set` for `ExtData%SNOWHGT` to read data from the GMAO `SNOMAS` field instead of `SNOWHGT`
```

Comment:
See my commend in `hcox_state_mod.F90`. It would make sense to me to rename the field to `ExtData%SNOMAS` throughout the code and documentation to avoid confusion. If agreed, we can create a separate feature request and PR.